module_utils/certificate_lsr/providers/base.py

```diff
@@ -672,6 +676,14 @@ def _write_param_to_file_if_diff(self, param_name, filepath, check_mode):
 
         # if file and param are the same just return
         if param_sha1 == file_sha1:
+            # Even if file and param are identical,
+            # if the script exists and the content of the script
+            # is different from the current one including the
+            # ansible managed comment, update the script.
+            cur_script = open(filepath, encoding="utf-8").read()
+            if param != cur_script:
+                with open(filepath, "w") as script_fp:
+                    script_fp.write(param)
```

Comment:
What happens if the only change made is a change in the ansible_managed value?  Will the file be updated, and the module report `changed: true`?

Reply:
You are right, @richm. I should have return `True` here. Thanks!